ci/jenkins/test-mc.sh

```diff
@@ -0,0 +1,300 @@
+WORKDIR=$DEFAULT_WORKDIR
+TESTCASE=""
+TEST_FAILURE=false
+DOCKER_REGISTRY=$(head -n1 "${WORKSPACE}/ci/docker-registry")
+GO_VERSION=$(head -n1 "${WORKSPACE}/build/images/deps/go-version")
+IMAGE_PULL_POLICY="Always"
+MULTICLUSTER_KUBECONFIG_PATH=$WORKDIR/.kube
+LEADER_CLUSTER_CONFIG="--kubeconfig=$MULTICLUSTER_KUBECONFIG_PATH/leader"
+EAST_CLUSTER_CONFIG="--kubeconfig=$MULTICLUSTER_KUBECONFIG_PATH/east"
+WEST_CLUSTER_CONFIG="--kubeconfig=$MULTICLUSTER_KUBECONFIG_PATH/west"
+
+CONTROL_PLANE_NODE_ROLE="control-plane,master"
+
+multicluster_kubeconfigs=($EAST_CLUSTER_CONFIG $LEADER_CLUSTER_CONFIG $WEST_CLUSTER_CONFIG)
+membercluter_kubeconfigs=($EAST_CLUSTER_CONFIG $WEST_CLUSTER_CONFIG)
+
+
+CLEAN_STALE_IMAGES="docker system prune --force --all --filter until=48h"
+
+_usage="Usage: $0 [--kubeconfigs-path <KubeconfigSavePath>] [--workdir <HomePath>]
+                  [--testcase <e2e>]
+
+Run Antrea multi-cluster e2e tests on a remote (Jenkins) Linux Cluster Set.
+
+        --kubeconfigs-path            Path of cluster set kubeconfigs.
+        --workdir                     Home path for Go, vSphere information and antrea_logs during cluster setup. Default is $WORKDIR.
+        --testcase                    Antrea multi-cluster e2e testcases on a Linux clusterset.
+        --registry                    The docker registry to use instead of dockerhub."
+
+function print_usage {
+    echoerr "$_usage"
+}
+
+
+while [[ $# -gt 0 ]]
+do
+key="$1"
+
+case $key in
+    --kubeconfigs-path)
+    MULTICLUSTER_KUBECONFIG_PATH="$2"
+    shift 2
+    ;;
+    --workdir)
+    WORKDIR="$2"
+    shift 2
+    ;;
+    --testcase)
+    TESTCASE="$2"
+    shift 2
+    ;;
+    --registry)
+    DOCKER_REGISTRY="$2"
+    shift 2
+    ;;
+    -h|--help)
+    print_usage
+    exit 0
+    ;;
+    *)    # unknown option
+    echoerr "Unknown option $1"
+    exit 1
+    ;;
+esac
+done
+
+function clean_tmp() {
+    echo "===== Clean up stale files & folders older than 7 days under /tmp ====="
+    CLEAN_LIST=(
+        "*codecov*"
+        "kustomize-*"
+        "*antrea*"
+        "go-build*"
+    )
+    for item in "${CLEAN_LIST[@]}"; do
+        find /tmp -name "${item}" -mtime +7 -exec rm -rf {} \; 2>&1 | grep -v "Permission denied" || true
+    done
+    find ${WORKDIR} -name "support-bundles*" -mtime +7 -exec rm -rf {} \; 2>&1 | grep -v "Permission denied" || true
+}
+
+
+function cleanup_multicluster_ns {
+    ns=$1
+    kubeconfig=$2
+
+    kubectl get pod -n "${ns}" --no-headers=true ${kubeconfig}| awk '{print $1}' | while read pod_name; do
+        kubectl delete pod "${pod_name}" -n "${ns}" --force --grace-period 0 ${kubeconfig} --timeout=30s
+    done
+    kubectl get svc -n "${ns}" --no-headers=true ${kubeconfig}| awk '{print $1}' | while read svc_name; do
+        kubectl delete svc "${svc_name}" -n "${ns}" --force --grace-period 0 ${kubeconfig} --timeout=30s
+    done
+    kubectl delete ns "${ns}" --ignore-not-found=true ${kubeconfig} --timeout=30s || true
```

Comment:
I think you can simply delete the namespace `${ns}`, all resources belongs to it will be deleted.